doc/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# KUKSA.val Quickstart
+
+The quickest possible way to get KUKSA.val up and running
+
+## Starting broker
+First we want to run KUKSA.val databroker
+
+```
+docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/databroker:master 
+```
+
+
+## Reading and Writing VSS data via CLI
+You can interact with the VSS datapoints using the cli clients. The first option is databroker-cli.
+
+This is, how you start it:
+
+```
+docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/databroker-cli:master       
+```
+
+Here is how you can use it:
+
+```
+client> get Vehicle.Speed 
+-> Vehicle.Speed: ( NotAvailable )
+client> feed Vehicle.Speed 200
+-> Ok
+client> get Vehicle.Speed 
+-> Vehicle.Speed: 200.00
+client> quit
+Bye bye!
+
+```
+
+An alternative is the kuksa-client CLI (based on our Python client library).
+
+Here is how you start it:
+
+```
+docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/kuksa-client:master --port 55555 --protocol grpc --insecure 
+```
+
+Here is how you can use it:
+
+
+```
+Test Client> getValue Vehicle.Speed
+{
+    "path": "Vehicle.Speed"
+}
+
+Test Client> setValue Vehicle.Speed 200
+OK
+
+Test Client> getValue Vehicle.Speed
+{
+    "path": "Vehicle.Speed",
+    "value": {
+        "value": 200.0,
+        "timestamp": "2023-01-16T12:43:57.305350+00:00"
+    }
+}
+
+Test Client> quit
+gRPC channel disconnected.
+
+```
+
+## Reading and Writing VSS data with code
+
+Most likely, to get productive with KUKSA.val you want to interact with it programatically. Here is you you do it with our Python library.
+
+### Generating data
+Create a file `speed_provider.py` with the following content
+
+```python
+from kuksa_client.grpc import VSSClient
+from kuksa_client.grpc import Datapoint
+
+import time
+
+with VSSClient('127.0.0.1', 55555) as client:
+    for speed in range(0,100):
+        client.set_current_values({
+        'Vehicle.Speed': Datapoint(speed),
+        })
+        print(f"Setting Vehicle.Speed to {speed}")
```

Comment:
```suggestion
        print(f"Feeding Vehicle.Speed to {speed}")
```
Maybe underline the meaning of Feed and Set. Only problem if someone just copies the code and uses not a sensor path. So I leave it to you :) Instead looks good to me.